Dataframe.lua

```diff
@@ -97,7 +97,13 @@ function Dataframe:load_table(...)
 		{arg='infer_schema', type='boolean', help='automatically detect columns\' type', default=true}
 	)
 
-	self.dataset = args.data
+	for k,v in pairs(args.data) do
+		if (type(v) ~= 'table') then
```

Comment:
With this, the column won't have the same length if it's not a table loaded. It would be better to add a filling value like NaN or zero